### PR TITLE
Move marker methods from prepare.go to query.go

### PR DESCRIPTION
**Pull request by Aflynn50**
Move the const `markerPrefix` and the methods `markerName` and `markerIndex` to `query.go`.

This change is motivated by PR #112. In it, the SQL will be generated in `query.go` so all calls to these functions will come from `query.go`.

### CODE REVIEW

**Review by niemeyer — CHANGES_REQUESTED** [review]
Let's please hold this until #112 itself has been approved, as the underlying assumption is that logic is moving from PrepareExpr to QueryExpr.

**Comment by Aflynn50** [comment]
Closed as this is covered by #123

**Pull request closed without merging**
